### PR TITLE
Final modular examples

### PR DESCRIPTION
**Pull request by victorwiebe**
Covering the last of the examples that are not modular, yet.

### CODE REVIEW

File: examples/vestingEscrowWallet.ts [hunk 1]
```diff
@@ -1,113 +1,79 @@
...
+  await addModule(
+    polymathAPI,
+    {
+      ticker,
+      moduleName: ModuleName.VestingEscrowWallet,
+    },
+    options,
+  );
+
+  await polymathAPI.moduleRegistry.subscribeAsync({
```
[the remark this answers is not in the document]
Comment (victorwiebe) [reply examples/vestingEscrowWallet.ts]: Can be removed

File: examples/usdTieredSTO.ts [hunk 2]
```diff
@@ -1,147 +1,96 @@
...
+      moduleName: ModuleName.UsdTieredSTO,
+    },
+    options,
+  );
 
-  const names: Promise<string>[] = [];
-  resultInstances.map(instanceFactory => {
-    names.push(instanceFactory.name());
+  await polymathAPI.moduleRegistry.subscribeAsync({
+    eventName: ModuleRegistryEvents_3_0_0.ModuleRegistered,
```
[the remark this answers is not in the document]
Comment (victorwiebe) [reply examples/usdTieredSTO.ts]: Can be removed

File: examples/vestingEscrowWallet.ts [hunk 3]
```diff
@@ -1,113 +1,79 @@
...
-
-  // Instantiate the API
-  const polymathAPI = new PolymathAPI(params);
-
-  // Get some poly tokens in your account and the security token
+import { PolymathAPI } from '../src/PolymathAPI';
+import { ModuleName } from '../src';
+import { VestingEscrowWalletData } from '../src/contract_wrappers/tokens/security_token_wrapper/common';
+import { AddingModuleOpts, addModule, moduleInstancesLookup } from './modules';
+import { ModuleRegistryEvents_3_0_0 } from '@polymathnetwork/abi-wrappers/lib/src';
```
[the remark this answers is not in the document]
Comment (victorwiebe) [reply examples/vestingEscrowWallet.ts]: Understood

File: examples/usdTieredSTO.ts [hunk 4]
```diff
@@ -1,11 +1,9 @@
 import { USDTieredSTOEvents_3_0_0, BigNumber } from '@polymathnetwork/abi-wrappers';
-import { PolymathAPI } from '../src/PolymathAPI';
-import { FundRaiseType, ModuleName } from '../src';
+import { FundRaiseType, ModuleName, PolymathAPI } from '../src';
```
Comment (monitz87) [inline examples/usdTieredSTO.ts]: These should be imported from their respective files (`FundRaiseType` and `ModuleName` from types.ts, `PolymathAPI` from PolymathAPI.ts)

File: examples/vestingEscrowWallet.ts [hunk 5]
```diff
@@ -1,9 +1,7 @@
 import { BigNumber } from '@polymathnetwork/abi-wrappers';
-import { PolymathAPI } from '../src/PolymathAPI';
-import { ModuleName } from '../src';
+import { PolymathAPI, ModuleName } from '../src';
```
Comment (monitz87) [inline examples/vestingEscrowWallet.ts]: Should be imported from their respective files

**Pull request merged**